charts/nextcloud/values.yaml

```diff
@@ -568,7 +566,8 @@ cronjob:
 
   # Allow configuration of lifecycle hooks
   # ref: https://kubernetes.io/docs/tasks/configure-pod-container/attach-handler-lifecycle-event/
-  lifecycle: {}
+  lifecycle:
+    {}
```

Comment:
Could you also return this one to the same standard please, sorry for the hassle.